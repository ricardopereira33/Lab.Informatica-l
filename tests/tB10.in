bcbcD
aAcbB
bBcaa
3 0 N
SEASLEAADSAL