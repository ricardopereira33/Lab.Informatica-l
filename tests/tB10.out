3 1 N
